src/routeApp/validations.js

```diff
@@ -0,0 +1,21 @@
+import { body } from 'express-validator';
+
+const routeValidation = () => [
+	// body('code', 'Code is required').notEmpty().toUpperCase(),
+	body('origin', 'Origin is required ').notEmpty(),
+	body('destination', 'Destination is required ').notEmpty(),
+	body('distance', 'This should not be empty').optional().isLength({
```

Comment:
Since this is a number/ integer in the database. Remember to sanitize the value with the `.toNumber()` and the validate accordingly. As we discussed we should not use `min length` on number but we use the `great or ` validation